packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -124,6 +138,19 @@ Each LogEvent object describes a single logged action or "event" that is perform
      "device": String, Optional
      "id": String, Optional
 },
+"device": { Object, Optional
```

Comment:
This comment is not related to this line. I just can't comment on the proper line.

There no indent on the top level fields on `LogEvent object annotated example` object.

Compare
![image](https://github.com/okta/okta-developer-docs/assets/32918538/542ed0e7-3a42-40d1-9960-4c5ef68536ad)
with
![image](https://github.com/okta/okta-developer-docs/assets/32918538/016af022-0970-4770-b46a-8e033badd877)

Reply:
Indent on `target` array is not correct

![image](https://github.com/okta/okta-developer-docs/assets/32918538/cffbde74-3603-481a-ab8a-db4c3bb7144a)

The indent on object opening bracket should be 4 spaces instead of 8. Basically, on the same level with the closing bracket of the object. As a result all the object fields should be less indented.

Reply:
`actor.detailEntry` contents should be indented

![image](https://github.com/okta/okta-developer-docs/assets/32918538/530c536f-7db7-4d72-b877-6bfd13673c73)

like in `target` object example

![image](https://github.com/okta/okta-developer-docs/assets/32918538/1c9ff8d5-2afa-4294-91e6-51688b200d82)

Reply:
A space is needed after `lat`
![image](https://github.com/okta/okta-developer-docs/assets/32918538/fbe356c0-f7f7-443b-b6eb-094d15f0fafc)

Reply:
It's not clear why `detailEntry` field is separated with `=` instead of `:`. It's like that on all the `detailEntry` fields.
![image](https://github.com/okta/okta-developer-docs/assets/32918538/0d106dc4-f527-4790-a729-2c8e17ba8437)

Reply:
`issuer` field metadata (Object, Optional) should be after the closing bracket. Like in `authenticationContext`

![image](https://github.com/okta/okta-developer-docs/assets/32918538/18df3a1c-0e49-497d-aec0-76e7c48f9fe3)

Reply:
Only `authenticationProvider` of `authenticationContext` has a proper indent. All the other fields of the object are shifted to the right.

![image](https://github.com/okta/okta-developer-docs/assets/32918538/b8f46db4-3e69-4c8a-9179-8ea0deb882e7)

Reply:
All fields of the last 3 objects should be less indented to be consistent with the the overall indent in this object.
![image](https://github.com/okta/okta-developer-docs/assets/32918538/687792c7-5ccc-4a56-b654-76015d91eb2e)

Reply:
ipChain is an `array` of `objects`

![image](https://github.com/okta/okta-developer-docs/assets/32918538/114d9633-5706-4acd-856a-0564cf5bc7b9)

There should be an opening bracket before the `ip` field, and a corresponding object closing bracket.
![image](https://github.com/okta/okta-developer-docs/assets/32918538/d4a45349-1580-4e36-9173-cc5275827153)